test/parallel/test-spawn-cmd-named-pipe.js

```diff
@@ -42,10 +42,8 @@ if (!process.argv[2]) {
     assert.fail('Failed to get COMSPEC');
   }
 
-  const args = ['/c', process.execPath, __filename, 'child',
-                '<', stdinPipeName, '>', stdoutPipeName];
-
-  const child = spawn(comspec, args);
+  const child = exec(`"${process.execPath}" "${__filename}" child < ${
```

Comment:
I've found out a way with `spawn()`:
```js
  const args = ['/c', `""${process.execPath}"`, `"${__filename}"`, 'child',
                '<', stdinPipeName, '>', `${stdoutPipeName}"`];

  const child = spawn(comspec, args, { shell: 'cmd.exe' });
```
But it seems strange. And we execute cmd.exe with cmd.exe first, if I get this rigtht. Id this way better?